assets/blocks/button/button-props.js

```diff
@@ -60,3 +64,12 @@ export function getButtonWrapperProps( { className, attributes: { align } } ) {
 		),
 	};
 }
+
+/**
+ * Check if block has the 'Link' block style.
+ *
+ * @param {Array} props Block props.
```

Comment:
I think it's an `Object` instead of an `Array`, right?

```suggestion
 * @param {Object} props Block props.
```

Reply:
Right, also fixed in d74b1b6